funnel/static/js/schedules.js

```diff
@@ -515,7 +515,9 @@ $(function () {
         buttons.save = (function () {
           calendar.container
             .find('.fc-header-right')
-            .append('<span class="hg-fc-button save-schedule">Save</span>');
+            .append(
+              '<span class="hg-fc-button mui-btn save-schedule">Save</span>'
```

Comment:
Translation missing.

Reply:
Updated babel.cfg to extract from .js files.